### PR TITLE
Internal: Fix plugin tester [ED-11872]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: tests/plugins-tester/.wp-env.json [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-    "themes": [
-        "https://downloads.wordpress.org/theme/hello-elementor.zip"
-    ],
-    "mappings": {
-        "elementor-templates": "./elementor-templates",
-        "scripts": "./scripts"
-    },
-    "port": 7777,
-    "testsPort": 7778
+	"phpVersion": "7.4",
```
Comment (mykytamurzin) [inline tests/plugins-tester/.wp-env.json:2]: 7.4 isn't supported [anymore](https://www.php.net/supported-versions.php). Do we consider switching to more fresh version?

Comment (dmaziar) [reply tests/plugins-tester/.wp-env.json:2]: We use the same for playwright. I'm not the expert here.

Comment (TzviRabinovitch) [reply tests/plugins-tester/.wp-env.json:2]: @mykytamurzin 
@dmaziar 
we support 7.0+ versions

**Review by TzviRabinovitch — APPROVED** [review]

File: tests/plugins-tester/.wp-env.json (hunk 1 above)
Comment (dmaziar) [reply tests/plugins-tester/.wp-env.json:2]: version 8.2 causes a lot of PHP Deprecated warnings. If we consider changing the version, it should be in a separate PR

**Pull request merged**